package.json

```diff
@@ -6,7 +6,6 @@
   "private": false,
   "dependencies": {
     "@dhis2/app-runtime": "^2.6.1",
-    "@dhis2/cli-utils-cypress": "^2.2.2",
```

Comment:
I must have missed it when installing the package but I move this to `devDependencies` now